test/e2e/create_test.go

```diff
@@ -304,6 +312,20 @@ func (tc *testContext) setPowerShellDefaultShell(machine *mapi.Machine) error {
 	})
 }
 
+// uninstallWindowsDefender uninstalls Windows Defender from the given machine. Reboots the underlying VM.
+func (tc *testContext) uninstallWindowsDefender(machine *mapi.Machine) error {
+	addr, err := controllers.GetAddress(machine.Status.Addresses)
+	if err != nil {
+		return err
+	}
+	command := "Uninstall-WindowsFeature -Name Windows-Defender; Restart-Computer -Force"
+	_, err = tc.runPowerShellSSHJob("uninstall-windows-defender", command, addr)
+	if err != nil {
+		return errors.Wrapf(err, "failed to uninstall Windows Defender antivirus", machine.GetName())
```

Comment:
missed variable removal here
could do errors.Wrap as well if you want

Reply:
Yep thanks, fixed